lms/djangoapps/discussion_api/tests/test_api.py

```diff
@@ -1804,7 +1804,7 @@ def test_discussions_disabled(self):
         )
         with self.assertRaises(ValidationError) as assertion:
             create_comment(self.request, self.minimal_data)
-        self.assertEqual(assertion.exception.message_dict, {"thread_id": ["Invalid value."]})
+        self.assertEqual(assertion.exception.message_dict, {"course_id": ["Invalid value."]})
```

Comment:
Can we have a better error message in this case?  The `course_id` is still valid, correct?  Just the discussions feature is disabled in the course?